be/src/exec/pipeline/fragment_context.cpp

```diff
@@ -317,4 +317,21 @@ void FragmentContext::count_down_epoch_pipeline(RuntimeState* state, size_t val)
     state->query_ctx()->stream_epoch_manager()->count_down_fragment_ctx(state, this);
 }
 
+void FragmentContext::init_jit_profile() {
+    if (runtime_state() && runtime_state()->is_jit_enabled() && runtime_state()->runtime_profile()) {
```

Comment:
suggest to 
```cpp
DCHECK(runtime_state()  && runtime_state()->runtime_profile());
if (runtime_state()->is_jit_enabled()) {
...
}
```
use  this condition must be ensured, if not so, A big catastrophe rather than jit not works would happen

Reply:
it would be null in be tests.